pkg/dns/dns.go

```diff
@@ -61,37 +64,46 @@ func (c *Config) Validate() error {
 	return fmt.Errorf("invalid DNS provider %q", c.Provider)
 }
 
-// AskToConfigure reads the required DNS entries from a Terraform output,
-// asks the user to configure them and checks if the configuration is correct.
-func (c *Config) AskToConfigure(ex *terraform.Executor) error {
-	dnsEntries, err := readDNSEntries(ex)
-	if err != nil {
-		return err
-	}
+// ManualConfigPrompt returns a terraform.ExecutionHook which prompts the user to configure DNS
+// entries manually and verifies the entries were created successfully.
+func ManualConfigPrompt(c *Config) terraform.ExecutionHook {
+	return func(ex *terraform.Executor) error {
+		dnsEntries, err := readDNSEntries(ex)
+		if err != nil {
+			return err
+		}
 
-	fmt.Printf("Please configure the following DNS entries at the DNS provider which hosts %q:\n", c.Zone)
-	prettyPrintDNSEntries(dnsEntries)
+		fmt.Printf("Please configure the following DNS entries at the DNS provider which hosts %q:\n", c.Zone)
+		prettyPrintDNSEntries(dnsEntries)
 
-	for {
-		fmt.Printf("Press Enter to check the entries or type \"skip\" to continue the installation: ")
+		for {
+			fmt.Printf("Press Enter to check the entries or type \"skip\" to continue the installation: ")
 
-		var input string
-		fmt.Scanln(&input)
+			var input string
 
-		if input == "skip" {
-			break
-		} else if input != "" {
-			continue
-		}
+			reader := bufio.NewReader(os.Stdin)
```

Comment:
It seems switch to NewReader should be done in separate commit?

Reply:
I did this to satisfy the linter. We didn't actually touch this code in this PR, we just moved it around a bit but it was enough for the linter to complain. The complaint was valid though - we had an unhandled error from `fmt.Scanln()`.

Does it still make sense to you to split this?

Reply:
Moved this to a separate commit to save time. I can re-squash if necessary.

Reply:
Ah, I didn't know the change was driven by the linter. I think it make sense to do that in separate commit then, thanks!